src/server/controllers/edit.controller.test.js

```diff
@@ -108,40 +100,6 @@ describe("Edit controller: editContinue()", () => {
           expect(result.cumulativeEditAnswers).toEqual(expectedAnswers);
         });
       });
-
-      describe("given the current page has checkboxes, one of which was originally checked but now is not", () => {
```

Comment:
this is a gernal test for the edit controller which just so happens to use this page. Please reinstate and update to use a smilar page that still exists